package.json

```diff
@@ -9,7 +9,7 @@
     "generate:types": "didc bind ./src/internet_identity/internet_identity.did -t ts > src/frontend/generated/internet_identity_types.d.ts",
     "generate:js": "didc bind ./src/internet_identity/internet_identity.did -t js > src/frontend/generated/internet_identity_idl.js",
     "build": "NODE_ENV=production webpack",
-    "test": "jest --roots ./src/frontend --verbose --testPathIgnorePatterns=\"src/frontend/src/test-e2e\"",
+    "test": "jest --maxWorkers 1 --roots ./src/frontend --verbose --testPathIgnorePatterns=\"src/frontend/src/test-e2e\"",
```

Comment:
Note: `BigInt` cannot by serialized to JSON by the default json encoder, and jest workers talk to each other using JSON. This means that a test failure involving a BigInt (for instance got `12n` when `42n` was expected) will throw a cryptic error about BigInt not being serializable. In order to avoid that, we set the number of workers to 1 so that no JSON communication is necessary. Unfortunately can't comment in the `package.json`. Alternative is to add the option to the jest config, but then it'd apply to all tests, which we may not want.